include/musica/util.hpp

```diff
@@ -92,4 +96,5 @@ bool operator==(const Error& lhs, const Error& rhs);
 /// @param rhs The right-hand side Error
 /// @return True if the Errors are not equal, false otherwise
 bool operator!=(const Error& lhs, const Error& rhs);
```

Comment:
If these functions are not inside the `#ifdef __cplusplus` block, I don't think a `.c` file could include this header because some of the arguments of these functions are references

Reply:
Ah, true. I fixed it.